Modules/QtWidgets/src/QmitkIOUtil.cpp

```diff
@@ -39,6 +39,40 @@ See LICENSE.txt or http://www.mitk.org for details.
 
 #include <algorithm>
 
+namespace
+{
+#if defined(_WIN32) || defined(_WIN64)
+    QString s_InvalidFilenameCharacters("<>:\"/\\|?*");
```

Comment:
Would prefer class member in Impl below.

Reply:
Thanks for your review in general! I implemented all of that. I did not mark the discussions resolved because I'd like you to verify if my changes are as asked for..